x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -433,6 +433,8 @@ export const RULES_TABLE_PAGE_SIZE_OPTIONS = [5, 10, 20, 50, RULES_TABLE_MAX_PAG
 export const RULES_MANAGEMENT_FEATURE_TOUR_STORAGE_KEY =
   'securitySolution.rulesManagementPage.newFeaturesTour.v8.2';
 
+export const NEW_TERMS_TOUR_ACTIVE_KEY = 'securitySolution.rulesManagementPage.newTermsTourActive';
```

Comment:
Is the reason for creating a new component and key for tour, instead of using existing one, issue with Eui tour mentioned in slack? Was there any feedback from Eui team?

If it is true, I think it would make sense to add comment regarding it to component and update README to existing component.

Reply:
I made it a separate component and tour because we didn't have any ask to show the old tour in addition to this one. If it's better though I can just replace the old tour steps with the new step, it seems we don't have plans to bring back the old tour?

Reply:
> If it's better though I can just replace the old tour steps with the new step

Yup, that would be a preferable way @marshallmain if you have time for that, because the idea was to update the existing tour that we have for the Rule Management page every release when we need to highlight some new features.

The existing tour is [here](https://github.com/banderror/kibana/blob/d30f367eb3498e1a3d3112296342023ddd3c7272/x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/feature_tour). You can just change its implementation to the one you need and update the local storage key to:

```ts
export const RULES_MANAGEMENT_FEATURE_TOUR_STORAGE_KEY =
  'securitySolution.rulesManagementPage.newFeaturesTour.v8.4';
```

Reply:
If you need help, I can help!